src/pages/iou/steps/MoneyRequestConfirmPage.js

```diff
@@ -423,11 +446,14 @@ export default compose(
         selectedTab: {
             key: `${ONYXKEYS.COLLECTION.SELECTED_TAB}${CONST.TAB.RECEIPT_TAB_ID}`,
         },
-    }),
-    // eslint-disable-next-line rulesdir/no-multiple-onyx-in-file
-    withOnyx({
         policy: {
             key: ({report}) => `${ONYXKEYS.COLLECTION.POLICY}${report ? report.policyID : '0'}`,
         },
+        policyCategories: {
+            key: ({policy}) => `${ONYXKEYS.COLLECTION.POLICY_CATEGORIES}${policy ? policy.id : '0'}`,
```

Comment:
So it looks that when this file is converted to TS that the `policy` will be of the `OnyxEntry` type which _could_  return null so this was just to go ahead and account for that. Once it is converted it can easily just be replaced with optional chaining like `policy?.id`.

Reply:
I asked about the multiple `withOnyx` thing from the comment above [on slack](https://expensify.slack.com/archives/C01GTK53T8Q/p1702335671208129)


> So it looks that when this file is converted to TS that the `policy` will be of the `OnyxEntry` type which _could_ return null so this was just to go ahead and account for that

But is it possible that this is because these are no longer defined in a separate `withOnyx`?